compiler/lsp-tests/src/Daml/Lsp/Test/Util.hs

```diff
@@ -0,0 +1,64 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE OverloadedStrings #-}
+module Daml.Lsp.Test.Util
+    ( Cursor
+    , cursorPosition
+    , expectDiagnostics
+    , damlId
+    , replaceDoc
+    ) where
+
+import Control.Applicative.Combinators
+import Control.Lens
+import Control.Monad
+import Control.Monad.IO.Class
+import qualified Data.Text as T
+import Language.Haskell.LSP.Test hiding (message)
+import qualified Language.Haskell.LSP.Test as LspTest
+import Language.Haskell.LSP.Types
+import Language.Haskell.LSP.Types.Lens
+import Test.Tasty.HUnit
+
+import DA.Test.Util
+
+-- | Convenient grouping of file path, 0-based line number, 0-based column number.
```

Comment:
This comment does not reflect the actual type since there's no file path.

Reply:
Good catch, removed. Also just to give some context: We definitely do want file paths here but `lsp-test` doesn’t expose them in a convenient way so for the initial attempt I’m not checking file paths and I’ll try to get this changed upstream.